clang/test/3C/arrinstructboth.c

```diff
@@ -123,26 +123,27 @@ z += 2;
 return z; }
 
 struct warr * foo() {
-	//CHECK_NOALL: struct warr * foo(void) {
-	//CHECK_ALL: _Ptr<struct warr> foo(void) {
+	//CHECK: _Ptr<struct warr> foo(void) {
         struct warr * x = malloc(sizeof(struct warr));
-	//CHECK: struct warr * x = malloc<struct warr>(sizeof(struct warr));
+	//CHECK: _Ptr<struct warr> x = malloc<struct warr>(sizeof(struct warr));
```

Comment:
This is less a review-comment and more because I'm curious: what changed in the code to allow line 127 to become checked? I think before, it was the case that this was marked WILD because it flows into `sus`, where the first parameter is set to 5. 
I think converting is the right thing to do here, since C is call-by-value and since doing `x = (void *) 5` in `sus` effectively separates the `x` from `sus` and the `x` from `foo`, which means it makes sense that one of them can be checked and the other can't, but wanted to bring this to your attention in case you think this is suspicious.